PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,19 +1,30 @@
-Please make sure you've read the [pull request](http://docs.mattermost.com/developer/contribution-guide.html#preparing-a-pull-request) section of our [code contribution guidelines](http://docs.mattermost.com/developer/contribution-guide.html).
+<!-- Thanks contribution a pull request! Here are some tips for you:
 
-When filling in a section please remove the help text and the above text.
+1. If this is your first contribution, make sure you've read the Contribution Checklist https://developers.mattermost.com/contribute/getting-started/contribution-checklist/
+2. Read our blog post about "Submitting Great PRs" https://developers.mattermost.com/blog/2019-01-24-submitting-great-prs
+3. Take a look at other repository specific documentation at https://developers.mattermost.com/contribute
+-->
 
 #### Summary
-[A brief description of what this pull request does.]
+<!--
+A description of what this pull request does.
+-->
 
 #### Ticket Link
-[Please link the GitHub issue or Jira ticket this PR addresses.]
+<!--
+Please link the GitHub issue, if this PR addresses a Help Wanted ticket e.g.
```

Comment:
Propose

```
If this pull request addresses a Help Wanted ticket, please link the relevant GitHub issue, e.g.

     Fixes https://github.com/mattermost/mattermost-server/issues/XXXXX

Otherwise, link the Jira ticket.
```